src/logging/timer.h

```diff
@@ -60,8 +60,8 @@ class Timer
 
     std::string LogMsg(const std::string& msg)
     {
-        const auto end_time = GetTime<std::chrono::microseconds>() - m_start_t;
-        if (m_start_t.count() <= 0) {
+        const auto end_time{std::chrono::duration_cast<std::chrono::microseconds>(std::chrono::steady_clock::now() - m_start_t)};
+        if (m_start_t == decltype(m_start_t){}) {
```

Comment:
I think that if statement will never evaluate to true, right? `m_start_t` is always initialized with `m_start_t = std::chrono::steady_clock::now();` so it shouldn't ever be zero.

Reply:
Huh? `m_start_t` is set *after* the first call to `LogMsg`, so I think this can evaluate to true.

Reply:
you're right, no idea why I repeatedly didn't see that call in the ctor...